spec/mailers/user_mailer_spec.rb

```diff
@@ -381,4 +381,55 @@
       end
     end
   end
+
+  describe "batch_subscription_notification" do
+    let(:work) { create(:posted_work, summary: "<p>Paragraph <u>one</u>.</p><p>Paragraph 2.</p>") }
```

Comment:
Tests are failing because of the renamed factory:
```
     # --- Caused by: ---
     # KeyError:
     #   key not found: "posted_work"
     #   ./spec/mailers/user_mailer_spec.rb:386:in `block (3 levels) in <top (required)>'
```

Reply:
Thanks!

Looking at the next line in this test, I wonder if we should do the same thing with chapters -- that is, make them posted by default.

Reply:
We should ([AO3-5968](https://otwarchive.atlassian.net/browse/AO3-5968)).

Reply:
❤️ ❤️ ❤️